do_everything.sh

```diff
@@ -357,3 +376,5 @@ echo '    lib/       contains static libraries'
 echo '  roscpp_android_ndk/     is a NDK sub-project that can be imported into an NDK app'
 echo '  sample_app/  is an example of such an app, a native activity'
```

Comment:
Can we be a tiny bit more explicit about which app is which? Maybe add a pointer to the corresponding ROS Wiki entries?
I mean so that the explanation of each of the two sample apps are not identical here.